test/js/css-image-value.js

```diff
@@ -0,0 +1,34 @@
+suite('CSSImageValue', function() {
+  test('CSSImageValue object is a CSSImageValue, CSSResourceValue, and CSSStyleValue', function() {
+    assert.instanceOf(new typedOM.internal.CSSImageValue(new Image()), typedOM.internal.CSSImageValue);
```

Comment:
Note that if developers are going to check something using instanceof, they don't have access to the internal classes. You may need to rearrange your function hiding thing to be the other way around.